### DISCUSSION TITLE
horizontal timeline in quasar

### DISCUSSION THREAD

**Discussion by bagladivyang03** [quasarframework/quasar#9441] [CLI - PWA mode]
Hey all,
How to use the horizontal timelines in quasar 
I saw the documentation and timeline component which is vertical only.
Can anyone pls help :)

**Comment by ontwikkelfabriek** [comment]
@bagladivyang03 

Quasar's timeline does not have horizontal settings. You could do this:
https://codepen.io/ontwikkelfabriek/pen/NWpvJdJ

But this Vue timeline component is horizontal:
https://vue-horizontal-timeline.netlify.app/?path=/story/vuehorizontaltimeline--default
https://github.com/guastallaigor/vue-horizontal-timeline
